djlsp/scripts/django-collector.py

```diff
@@ -370,7 +370,12 @@ def _get_template(engine: Engine, template_name: str) -> Template:
 
 
 def get_global_template_context():
-    global_context = {}
+    global_context = {
+        # builtins
+        "True": None,
+        "False": None,
+        "None": None,
```

Comment:
What is the purpose of setting these values to `None` here?

Reply:
Because it has no type completion